### PR TITLE
168690665 users should be able to register.

### PR DESCRIPTION
**Pull request by anyatibrian**
###  What does this PR do?  ###

- enables various users to create accounts 

### Description of tasks to be completed ###

- add an endpoint for creating new users
- add validation during the creation of users
- add test for a various endpoint edge cases

add the comments components to the feature
###  How can this be tested? ###
Clone the repo `https://github.com/anyatibrian/IRent-backend.git`
Checkout `ft-signup-168690665`
`yarn install` or `npm install` and to run the application `run yarn or npm run dev`

### Relevant PT board story ###
[#168690665](https://www.pivotaltracker.com/story/show/168690665)
###  Postman collection  ###

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import express, { Request, Response } from 'express'
+import userRouter from './resources/userAuth/user.routes'
+import { errors } from 'celebrate'
 import morgan from 'morgan'
 
 /** @params {string} express function */
+const prefix = '/api/v1/'
 
 export const app = express()
 export const port: any = process.env.PORT || 8080
```
Comment (MuhweziDeo) [inline src/app.ts]: Number  not any

File: src/services/db.services.ts [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
-class ServicesProvidyver {
+import { Op } from 'sequelize'
+class ServicesProvidver {
   /**
    * @params {body}
    * @returns {newUser}
    */
-  public createUser = async (body: any): Promise<any> => {
-    const newUser: any = await db.UserModel.create(body)
+  public static createUser = (body: any) => {
```
Comment (MuhweziDeo) [inline src/services/db.services.ts]: Avoid using any lets, create an interface for the userbody
